vagrant/provisioning/roles/arkcase-app/tasks/main.yml

```diff
@@ -911,6 +911,15 @@
     dest: /home/arkcase/.arkcase/acm/acm-config-server-repo/arkcase-commons-server.yaml
   when: arkcase_version == "" or arkcase_version is version('2021.04', '>=')
 
+- name: server-specific configuration for arkcase-portal-oidc.yml for version 2021.04
```

Comment:
As i am aware of, ArkCase is at 2022.01-RC1. Are we going with 2021.04?

Reply:
2021.04 is the convention used in other version checks for any version higher than 2021.03, including 2022 and future versions.

Reply:
Perfect than. Thanks for the clarification!.